db/schema.rb

```diff
@@ -0,0 +1,51 @@
+# rubocop:disable Metrics/BlockLength
+ActiveRecord::Schema[7.0].define(version: 20_221_209_131_446) do
+  # These are extensions that must be enabled in order to support this database schema
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.string 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'user_id', null: false
+    t.bigint 'post_id', null: false
+    t.index ['post_id'], name: 'index_comments_on_post_id'
+    t.index ['user_id'], name: 'index_comments_on_user_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'user_id', null: false
+    t.bigint 'post_id', null: false
+    t.index ['post_id'], name: 'index_likes_on_post_id'
+    t.index ['user_id'], name: 'index_likes_on_user_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.string 'text'
+    t.integer 'comments_counter'
+    t.integer 'likes_counter'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'user_id', null: false
```

Comment:
Please add `author_id` as a foreign key in `posts` table as it can be shown in this [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).